### PR TITLE
Update CHANGELOG for Firestore v0.12.4

### PR DESCRIPTION
**Pull request by wilhuff**

### CODE REVIEW

File: Firestore/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Unreleased
 
+# v0.12.4
+- [fixed] DocumentReference.setData(_:mergeFields:) can now delete fields using
+  FieldValue.delete().
+- [fixed] Firestore will now transparently recover from auth token expiration.
```
Comment (var-const) [inline Firestore/CHANGELOG.md]: Perhaps something like `Firestore will now recover from auth token expiration when the system clock is wrong`

File: Firestore/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Unreleased
 
+# v0.12.4
+- [fixed] DocumentReference.setData(_:mergeFields:) can now delete fields using
```
Comment (schmidt-sebastian) [inline Firestore/CHANGELOG.md]: Feel free to leave as is, but this applies to WriteBatch and Transaction as well.

File: Firestore/CHANGELOG.md (hunk 1 above)
Comment (wilhuff) [reply Firestore/CHANGELOG.md]: Done.

File: Firestore/CHANGELOG.md (hunk 2 above)
Comment (wilhuff) [reply Firestore/CHANGELOG.md]: Fixed.

**Pull request merged**